packages/tools/devtools/devtools-core/src/test/types/validateDevtoolsCorePrevious.generated.ts

```diff
@@ -94,6 +96,7 @@ declare type current_as_old_for_Interface_IDevtoolsLogger = requireAssignableTo<
  * typeValidation.broken:
  * "Interface_IFluidDevtools": {"forwardCompat": false}
  */
+// @ts-expect-error compatibility expected to be broken
```

Comment:
This backward incompatibility will break the office-bohemia because we recently check-in the "integrate devtools in loop" code change and that commit use `registerContainerDevtools` API.